cmd/neofs-node/config.go

```diff
@@ -969,8 +971,45 @@ func (c *cfg) configWatcher(ctx context.Context) {
 	}
 }
 
-// writeAppVersion writes app version as defined at compilation
-// step to the node's attributes.
-func writeAppVersion(c *cfg) {
+// writeSystemAttributes writes app version as defined at compilation
+// step to the node's attributes and an aggregated disk capacity
+// according to the all space on the all configured disks.
+func writeSystemAttributes(c *cfg) error {
+	// `Version` attribute
+
 	c.cfgNodeInfo.localInfo.SetAttribute("Version", misc.Version)
+
+	// `Capacity` attribute
+
+	var stat syscall.Statfs_t
+	var total uint64 // bytes
+	fsIDMap := make(map[syscall.Fsid]struct{})
+
+	for _, sh := range c.applicationConfiguration.EngineCfg.shards {
+		for _, storage := range sh.subStorages {
+			path := storage.path
+
+			err := util.MkdirAllX(path, storage.perm)
+			if err != nil {
+				return fmt.Errorf("can not create (ensure it exists) dir by '%s' path: %w", path, err)
+			}
+
+			err = syscall.Statfs(path, &stat)
+			if err != nil {
+				return fmt.Errorf("get FS stat by '%s' path: %w", path, err)
+			}
+
+			if _, handled := fsIDMap[stat.Fsid]; handled {
+				continue
+			}
+
+			total += stat.Blocks * uint64(stat.Bsize)
+			fsIDMap[stat.Fsid] = struct{}{}
+		}
+	}
+
+	// according to API description, in GB, not GiB
```

Comment:
Then it's obviously 1024^3. We do not use `iB` notation and likely won't be.

Reply:
so make `1 << 30` instead of `100*1000*1000`?

Reply:
Sure.